src/main/java/org/myrobotlab/service/interfaces/Simulator.java

```diff
@@ -14,13 +12,19 @@
  *
  */
 
+// FIXME !!! refactor this stuff out - create an AbstractSimulator ... 
 public interface Simulator extends NameProvider {

```

Comment:
Seems like this should implement ServoController ...

Reply:
i hope not .. 
Right now it "has a" ServoController but "is a" Simulator - and when I get MotorController working and all sorts of other virtualized sensors or control components - I don't want the simulator to "be a" "everything"